workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/ComponentVisualElement.cs

```diff
@@ -34,6 +35,18 @@ protected void InjectComponentIcon(string iconName)
             foldoutToggle.Insert(1, iconVisualElement);
         }
 
+        protected void AddInfoIcon(Action action)
```

Comment:
Nit: Rather than passing in a closure, we could add a `private abstract void WriteDebugInfo()` method which we register to the `clicked` property and implement in the child class.

Reply:
Where's the xml bit supposed to go?